frontend/src/styles/CreateMoment.css

```diff
@@ -1,31 +1,227 @@
+@import url('https://fonts.googleapis.com/css?family=Teko');
+
 .CreateMoment {
+    text-align: center;
+    font-size: 30px;
     margin-top: 20px;
-    margin-left: 20px;
+    margin-left: 40px;
     margin-right: auto;
     padding-left: 0px;
     padding-top: 0px;
     height: 20%;
 }
 
+div.createMoment-header{
+  text-align: center;
+  margin-top: 20px;
+  padding-bottom: 40px;
+}
+
+div.createMoment-header img{
+  text-align: center;
+  width: 420px;
+}
+
+div.createMoment-description {
+  display: block;
+  position: center;
+  text-align: center;
+  font-family: 'Teko';
+  color: #1DE28F;
+  font-size: 48px;
+}
 .content-container {
+    display: block;
+    position: center;
+    margin-top: 30px;
     margin-bottom: 20px;
-}
 
+}
 .label {
     margin-right: 40px;
 }
 
-.title { 
-    width: 300px;
-}
-
 .textarea {
     padding-left: 10px;
+
 }
 
 .button {
-    background-color: rgb(218, 222, 224);
-    font-size: 16px;
-    border-radius: 12px;
-    border-color: black;
-}
\ No newline at end of file
+    cursor: pointer;
+    background-color: rgb(255, 255, 255);
+    font-size: 26px;
+    font-family: 'Teko';
+    width: 90px;
+    border-radius: 2px;
+}
+.button:hover{
+    border-width: 2px;
+    border-color: #1DE28F;
+}
+/*
+.upload-btn-wrapper input[type=file] {
+  font-size: 100px;
+  position: absolute;
+  left: 0;
+  top: 0;
+  opacity: 0;
+}
+*/
+.inputfile {
+  cursor: pointer;
+	width: 0.1px;
+	height: 0.01px;
+	overflow: hidden;
+	position: absolute;
+	z-index: -1;
+  background-color: white;
+  font-weight: 200;
+  font-size: 40px;
+  border-radius: 3px;
+  border-color: white;
+  border-width: 2px;
+  display: inline-block;
+}
+.content-container img{
+  height:34px;
+  width: 35px;
+  position: relative;
+  top: 9.5px;
+  padding-left: 3px;
+}
+
+.inputfile + label {
+    position: center;
+    cursor: pointer;
+    font-size: 30px;
+    font-weight: 700;
+    height:50px;
+    width: 200px;
+    padding: 0px 10px 0px 10px;
+    font-family: 'Teko';
+    font-weight: 200;
+    text-align:center;
+    background-color: #FFFFFF;
+    border-radius: 3px;
+    border-width: thin;
+    border-color: #202020;
+    display: inline-block;
+    margin: 8px;
+}
+.inputfile:focus + label {
+	outline: 1px dotted #FFFFFF;
+	outline: -webkit-focus-ring-color auto 5px;
+}
+
+.inputfile:focus + label,
+.inputfile + label:hover {
+  border-width: 2px;
+  border-color: #1DE28F;
+}
+
+#filename {
+  font-family: 'Teko';
+  color: #FFFFFF;
+  font-size: 25px;
+  font-weight: 200;
+}
+
+:root {
+  /* generic */
+  --transitionTF: cubic-bezier(0.645, 0.045, 0.355, 1);
+  /* floated labels */
+  --labelTransformedPosY: calc(
+    (50%) -
+    (1rem * 0.8) -
+    (2.4rem * 1.8)
+  );
+  --inputTransitionTF: var(--transitionTF);
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
```

Comment:
Why define a fontsize for every element on the page when you're defining individual font sizes?